packages/builder/src/components/automation/AutomationPanel/AutomationNavItem.svelte

```diff
@@ -35,45 +35,53 @@
   }
 
   const getContextMenuItems = () => {
-    return [
-      {
-        icon: "Delete",
-        name: "Delete",
-        keyBind: null,
-        visible: true,
-        disabled: false,
-        callback: confirmDeleteDialog.show,
-      },
-      {
-        icon: "Edit",
-        name: "Edit",
-        keyBind: null,
-        visible: true,
-        disabled: false,
-        callback: updateAutomationDialog.show,
-      },
-      {
-        icon: "Duplicate",
-        name: "Duplicate",
-        keyBind: null,
-        visible: true,
-        disabled: automation.definition.trigger.name === "Webhook",
-        callback: duplicateAutomation,
-      },
-      {
-        icon: automation.disabled ? "CheckmarkCircle" : "Cancel",
-        name: automation.disabled ? "Activate" : "Pause",
-        keyBind: null,
-        visible: true,
-        disabled: false,
-        callback: () => {
-          automationStore.actions.toggleDisabled(
-            automation._id,
-            automation.disabled
-          )
-        },
+    const isRowAction = automation.definition.trigger.name === "Row Action"
+    const result = []
+    if (!isRowAction) {
+      result.push(
+        ...[
```

Comment:
How come this spread is needed? Could you not ditch the array entirely and pass the objects in bare?

Reply:
Yes, that would be the same than 3 separate `.push(object)`.